CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.8.11 (core) / 0.24.11 (libraries)
+
+### New
+
+- AutomationCondition.eager() will now only launch runs for missing partitions which become missing _after_ the condition has been added to the asset. This avoids situations in which the eager policy kicks off a large amount of work when added to an asset with many missing historical static/dynamic partitions.
```

Comment:
Ok I'm being annoying / sorry for missing this in the first pass but these first two should also be tagged with [experimental]

Reply:
no it's good you're reading carefully, thanks